core/src/components/card/test/theme-ionic/index.html

```diff
@@ -13,6 +13,13 @@
     <script src="../../../../../scripts/testing/scripts.js"></script>
     <script nomodule src="../../../../../dist/ionic/ionic.js"></script>
     <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+    <script>
```

Comment:
5 of the "theme-ionic" templates use `theme="ionic"` as an attribute on the `html` element. We should be consistent.